Jenkinsfile

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+final String COMPONENT = "nagios4"
+final String VERSION = "4.0.8"
```

Comment:
Does this version have to be kept in sync with a version anywhere else?

Reply:
This is the nagios version being installed, if that changes then this will need to be updated.